s2i/tests/resources.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: nodejs-s2i-example
+spec:
+  type: git
+  params:
+  - name: revision
+    value: master
+  - name: url
+    value: https://github.com/chmouel/nodejs-health-check
+---
+apiVersion: tekton.dev/v1alpha1
+kind: PipelineResource
+metadata:
+  name: image
+spec:
+  type: image
+  params:
+  - name: url
+    value: localhost:5000/nodejs-health-check-tekton
```

Comment:
it might be nice to [embed the pipelineresources](https://github.com/tektoncd/pipeline/blob/master/docs/taskruns.md#providing-resources) in the taskruns - would mean only one file / resource to look at to understand how the Task is used / how the test works

Reply:
ah i forgot this went in :-D 

thanks for pointing this out ill modify the tasks to use this,

Reply:
will work on this on another PR as an iteration improvement,